docker_shell.sh

```diff
@@ -103,28 +102,26 @@ function run_docker() {
 	$MOCK_AREA_TCL_PREFIXED \
 	-v $WORKSPACE_ROOT:$WORKSPACE_ROOT \
 	-v $WORKSPACE_ORIGIN:$WORKSPACE_ORIGIN \
+	-w $WORKSPACE_EXECROOT \
 	--network host \
 	$DOCKER_INTERACTIVE \
 	$DOCKER_ARGS \
 	${OR_IMAGE:-openroad/flow-ubuntu22.04-builder:latest} \
-	bash -c \
-	"set -e$DEBUG
```

Comment:
@hovind ?

Reply:
@lpawelcz thanks for having a look. I dropped the `bash` invocation entirely, so we have `docker run ... make ...` instead of `docker run ... bash -c "set -ex; make ..."`. What does the `bash` invocation achieve?